Added Login page to the project

AhmedGomaa15 opened the pull request:
### Description

- Added login page with its design.
- Changed the whole design (styles for App.css & global.css) of the project.
- Continue on the structure as we discussed  before 

### Issue
N/A

### Checklist
- [ ] Code has been tested locally and is ready for review.
- [ ] All automated tests pass.
- [ ] Documentation has been updated, if necessary.
- [ ] The code follows our coding standards.
- [ ] This feature has been discussed and approved by the team (if applicable).

### Reviewer Guidelines
Please review the following:
- [ ] Code quality and style.
- [ ] Testing and coverage.
- [ ] Documentation.
- [ ] Any potential security concerns.


### Screenshots (if applicable)
Add any relevant screenshots or images that can help with the review.

### Additional Notes
Any additional information or context that might be useful for the reviewer?

### Definition of Done
- [ ] Code has been reviewed and approved.
- [ ] All checks pass, and automated tests are green.
- [ ] Documentation is up-to-date.
- [ ] Merged without conflicts.
- [ ] Deployed to staging for further testing (if applicable).

### References
Add any relevant links, references, or related PRs.

AhmedGomaa15 committed:
test file includes hello

AhmedGomaa15 committed:
Description
edit the whole style and make login page.

Issue
N/A

Checklist
 Code has been tested locally and is ready for review.
 All automated tests pass.
 Documentation has been updated, if necessary.
 The code follows our coding standards.
 This feature has been discussed and approved by the team (if applicable).
Reviewer Guidelines
Please review the following:

 Code quality and style.
 Testing and coverage.
 Any potential security concerns.
Screenshots (if applicable)
Add any relevant screenshots or images that can help with the review.

Reviewers
@karimohamed60 @ayaeman.

Definition of Done
 Code has been reviewed and approved.
 All checks pass, and automated tests are green.
 Documentation is up-to-date.
 Merged without conflicts.
 Deployed to staging for further testing (if applicable).

@ index.html i changed the title into PHU
@ app.css & index.css i changed the whole style
@ login.tsx I added the form with styling to the login page as we discussed about their design before

AhmedGomaa15 committed:
- Make Login Component
- Added login page with its design.
- Changed the whole design (styles for App.css & global.css) of the project.
- Continue on the structure as we discussed  before

N/A

- [ ] Code has been tested locally and is ready for review.
- [ ] All automated tests pass.
- [ ] Documentation has been updated, if necessary.
- [ ] The code follows our coding standards.
- [ ] This feature has been discussed and approved by the team (if applicable).

Please review the following:
- [ ] Code quality and style.
- [ ] Testing and coverage.
- [ ] Documentation.
- [ ] Any potential security concerns.

Add any relevant screenshots or images that can help with the review.

Any additional information or context that might be useful for the reviewer?

- [ ] Code has been reviewed and approved.
- [ ] All checks pass, and automated tests are green.
- [ ] Documentation is up-to-date.
- [ ] Merged without conflicts.
- [ ] Deployed to staging for further testing (if applicable).

Add any relevant links, references, or related PRs.

AhmedGomaa15 committed:
- Added Pages folder
- Added login page with its design.
- Changed the whole design (styles for App.css & global.css) of the project.
- Continue on the structure as we discussed  before

N/A

- [ ] Code has been tested locally and is ready for review.
- [ ] All automated tests pass.
- [ ] Documentation has been updated, if necessary.
- [ ] The code follows our coding standards.
- [ ] This feature has been discussed and approved by the team (if applicable).

Please review the following:
- [ ] Code quality and style.
- [ ] Testing and coverage.
- [ ] Documentation.
- [ ] Any potential security concerns.

Add any relevant screenshots or images that can help with the review.

Any additional information or context that might be useful for the reviewer?

- [ ] Code has been reviewed and approved.
- [ ] All checks pass, and automated tests are green.
- [ ] Documentation is up-to-date.
- [ ] Merged without conflicts.
- [ ] Deployed to staging for further testing (if applicable).

Add any relevant links, references, or related PRs.

ayaeman commented on client/src/Pages/Login/Login.tsx:
```diff
@@ -0,0 +1,91 @@
+      <div className="col-12 col-md-8 col-lg-6 col-xl-5">
+        <div
+          className="card bg-white text-dark"
+          style={{ borderRadius: "1rem" }}
```
Remove style from Login.tsx to Login.css

AhmedGomaa15 committed:
- Added login page with its design.
- Changed the whole design (styles for App.css & global.css) of the project.
- Continue on the structure as we discussed  before

N/A

- [ ] Code has been tested locally and is ready for review.
- [ ] All automated tests pass.
- [ ] Documentation has been updated, if necessary.
- [ ] The code follows our coding standards.
- [ ] This feature has been discussed and approved by the team (if applicable).

Please review the following:
- [ ] Code quality and style.
- [ ] Testing and coverage.
- [ ] Documentation.
- [ ] Any potential security concerns.

Add any relevant screenshots or images that can help with the review.

Any additional information or context that might be useful for the reviewer?

- [ ] Code has been reviewed and approved.
- [ ] All checks pass, and automated tests are green.
- [ ] Documentation is up-to-date.
- [ ] Merged without conflicts.
- [ ] Deployed to staging for further testing (if applicable).

Add any relevant links, references, or related PRs.
